src/cli.rs

```diff
@@ -337,7 +351,7 @@ impl Cli {
                         Colorize::yellow(&*name),
                         Colorize::blue(&*rename.unwrap())
                     );
-                } else if add.is_some() {
+                } else if let Some(..) = add {
```

Comment:
If you don't need the `T` inside an `Option<T>`, it is recommended to use `is_some()`.

```suggestion
                } else if add.is_some() {
```